tests/x11/reboot_gnome.pm

```diff
@@ -40,12 +40,19 @@ sub run() {
 
         send_key "ret";
 
-        wait_still_screen 8, 8;    # wait 8 seconds to make authentication window disappear after successful authentication
```

Comment:
yes, exactly, but I didn't want to argue with @okurz about sleep :)
We have to wait a bit for window to disappear, to decide if reboot timed out or not, and it can't be too long otherwise the fastest test could miss grub
4 was too less e.g. on ppc64le, 8 seconds looks to be safe value